docs/open-source-projects.mdx

```diff
@@ -0,0 +1,538 @@
+	</a>
+</p>
+
+- **Website**: [React Components](https://components.eosio.cr)
+
+- **GitHub**: [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/eoscr-components)
+
+#### Page
+
+<p align="center">
+	<img src={useBaseUrl('img/OSS_screnshots/React.png')} width="100%"/>
+</p>
+
+<p align="center">
+	<img src={useBaseUrl('img/OSS_screnshots/React_2.png')} width="100%"/>
+</p>
+
+* * *
+
+## EOSIO Hackathons 
+
+### LifeBank
+
+<p align="left">
+	<a href="https://lifebank.io/">
+		<img src="https://raw.githubusercontent.com/eoscostarica/lifebank/master/docs/logos/2-OverWhite-lifebank-logo-v1-may25-2020-01.svg" width="40%"/>
+	</a>
+</p>
+
+ Lifebank is an EOSIO-based dapp that helps local communities create a virtuous circle of value exchange between three parties — eligible life donors, community donation centers, and participating local businesses (sponsors).
+
+ Lifebank is an initiative of EOS Costa Rica, a company that develops blockchain-based solutions to solve real-world problems. Lifebank is our entry to the Coding for Change challenge organized by leading blockchain company, Block.one.
+
+- **Website**: [Lifebank](https://lifebank.io/)
+
+- **GitHub**: [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/lifebank)
+
+#### LifeBank Page
+
+<p align="center">
+	<img src={useBaseUrl('img/OSS_screnshots/LifeBank.png')} width="100%"/>
+</p>
+
+<p align="center">
+	<img src={useBaseUrl('img/OSS_screnshots/LifeBank_2.png')} width="100%"/>
+</p>
+
+### gGoods
+
+<p align="left">
+	<a href="https://ggoods.io/">
+		<img src="https://raw.githubusercontent.com/eoscostarica/gGoods/main/docs/ggoods-logo.svg" width="40%"/>
+	</a>
+</p>
+
+ gGoods is an open-source NFT standard that enables organizations and communities to create their own NFTs to raise funds. Creating an NFT is easy and straightforward using our avatar creator. The NFTs are then purchased by donors as collectibles that not only support a cause but can be integrated into countless applications such as games, emojis, chat stickers, avatars etc.
+
+- **Website**: [gGoods](https://ggoods.io/)
+
+- **GitHub**: [<img src="https://cdn-icons-png.flaticon.com/32/25/25231.png"/>](https://github.com/eoscostarica/gGoods)
+
+#### gGoods Page
+
+<p align="center">
+	<img src={useBaseUrl('img/OSS_screnshots/gGoods.png')} width="100%"/>
+</p>
+
+* * *
+
+## NPM Packages
+
+<div class="table-wrapper" markdown="block">
+
+| Components | Standard Theme  | API module | Fastify EOS | 
+|:-------:|:-------:|:-------:|:-------:|
+|    A collection of React Components for EOSIO.    | EOS Costa Rica standard theme will allow us a faster developments of our web apps and at the same time it's availabe for anyone who will want to use it. |  This project wraps the official eosio/eosjs-api to provide camelcase output. | A plugin that decorates Fastify with an EOS.js instance. |
```

Comment:
Change `EOS Costa Rica standard theme will allow us a faster developments of our web apps` to `EOS Costa Rica standard theme allows us to speed up the development of our web apps`.